jest.config.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const jestConfig = require('./index');
+
+module.exports = jestConfig({
+    snapshotSerializers: [
+        'jest-serializer-path',
+    ],
+});
```

Comment:
Why do we need this file? Can't we use `snapshotSerializers` inside the exported config?

Reply:
I'm not sure what you mean. I can add the serializer to `jest-config.js`, making it a default of the package. 

But i think i'd still need this `jest.config.js` file, to define the test configuration for this package.

Reply:
I believe we shoud use a single `index.js` file that would export the configuration function, and configuration would already have `snapshotSerializers`. Since we're not using the JEST Presets, we don't have to follow any file naming convention.